rules/false_positives/ltp.yara

```diff
@@ -0,0 +1,16 @@
+rule dirty_pipe_test: override {
+  meta:
+    ELASTIC_Linux_Exploit_CVE_2022_0847_E831C285  = "low"
+    SIGNATURE_BASE_WEBSHELL_ASPX_Proxyshell_Aug15 = "low"
+
+  strings:
+    $comment1 = " * Ported into LTP by Yang Xu <xuyang2018.jy@fujitsu.com>"
```

Comment:
This false-positive feels very much like an exploit for 2022_0847. I don't think it should be a false-positive.

Reply:
It's an LTP test case: https://github.com/linux-test-project/ltp/blob/a790fadea35e65b46030f60e389deee7a6118c2b/testcases/kernel/security/dirtypipe/dirtypipe.c#L4

It is technically the actual exploit.

Reply:
Removed this override in [`94f75d4` (#838)](https://github.com/chainguard-dev/malcontent/pull/838/commits/94f75d4a846047ff3d98be23b379eb9d577c6407).